app/controllers/harvests_controller.rb

```diff
@@ -65,6 +72,8 @@ def create
   # PUT /harvests/1
   # PUT /harvests/1.json
   def update
+    planting_is_mine!
```

Comment:
I don't know why it doesn't check owner id on update. It's an allowed param. So I think it should.